src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/config/RestTemplateConfiguration.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.config;
+
+import org.springframework.boot.web.client.RestTemplateBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class RestTemplateConfiguration {
+
+    @Bean
+    public RestTemplate restTemplate(RestTemplateBuilder builder) {
+        return builder.build();
+    }
+
```

Comment:
Do we need the rest template thought we were using feign clients only

Reply:
this was addressed in a slack converstation